### PR TITLE
Merchant index

### PR DESCRIPTION
**Pull request by reid-andrew**
Code for user stories 38-41
- Admin ability to disable merchants & their items
- Admin ability to re-enable merchants & their items
- Includes one migration for adding an "active" status to merchants.

### CODE REVIEW

File: app/controllers/admin/merchants_controller.rb [hunk 1]
```diff
@@ -6,8 +6,37 @@ def require_admin
...
+      end
+      flash[:success] = "#{@merchant.name}'s account has been disabled."
+    elsif
+      merchant_params[:type] == "enable"
+      @merchant.update(:active => true)
+      @merchant.items.each do |item|
+        item.update(:active? => true)
+      end
+      flash[:success] = "#{@merchant.name}'s account has been enabled."
+    end
```
Comment (reid-andrew) [inline app/controllers/admin/merchants_controller.rb:28]: This method is good candidate for refactoring. We stopped at getting working code in the system.

Comment (javier-aguilar) [reply app/controllers/admin/merchants_controller.rb:28]: Agreed. We have a similar situation with our items controller

**Review by javier-aguilar — APPROVED** [review]
I didn't see anything that stood out.

**Review by mlynch5187 — APPROVED** [review]
Looking good, I saw nothing major to change. The update method in the admin merchants controller could be refactored at a later time.

**Pull request merged**